master.js

```diff
@@ -89,7 +89,7 @@ class AlAzureMaster {
         };
         this._apiEndpoint = alApiEndpoint ? alApiEndpoint : process.env.CUSTOMCONNSTR_APP_AL_API_ENDPOINT;
         this._alAzcollectEndpoint = alAzcollectEndpoint ? alAzcollectEndpoint : process.env.APP_AZCOLLECT_ENDPOINT;
-        this._aimsc = new alcollector.AimsC(this._apiEndpoint, creds, undefined, MASTER_RETRY_OPTS);
+        this._aimsc = new alcollector.AimsC(this._apiEndpoint, creds, process.env.TMP, MASTER_RETRY_OPTS);
```

Comment:
Should the AimsC constructor also use process.env.TMP instead of DEFAULT_CACHE_DIR ('/tmp')?

Reply:
I don't want to rely on `process.env` in `al-collector-js`. TMP directory path may come from configuration parameters of consumers. Going forward `al-aws-collector-js` and `al-azure-collector-js` use `process.env.TMP`